google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/datacatalog_facade.py

```diff
@@ -106,7 +110,18 @@ def upsert_entry(self, entry_group_name, entry_id, entry):
         return persisted_entry
 
     @classmethod
-    def __entry_was_updated(cls, current_entry, new_entry):
+    def __entry_was_updated(cls, *, current_entry, new_entry):
```

Comment:
Does the order actually matter for the comparison results? I'm not sure... 🤔

Reply:
Actually I did two optimizations, first to return early if the columns don't match... and second to initialize both lists with the default value of the GRPC client (when the entry is returned from the get_entry call for example and not built from the code).

So the order does not matter anymore, thanks! I will remove that requirement.